### PR TITLE
fix: remove initial ugs popup [MTT-3563]

### PR DESCRIPTION
**Pull request by LPLafontaineB**
### Description

This removes the initial popup telling users to setup UGS if it is not already setup. It also renames the buttons in the main menu from "Start" and "Direct IP" to "Start with Lobby" and "Start with Direct IP". The popup is replaced by a tooltip that shows when the cursor hovers on the disabled "Start with Lobby" button.

### Issue Number(s)

[MTT-3563](https://jira.unity3d.com/browse/MTT-3563)

### Contribution checklist
 - [ ] Tests have been added for boss room and/or utilities pack
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file and/or [package changelog](../Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] JIRA ticket ID is in the PR title or at least one commit message
 - [x] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

File: Assets/BossRoom/Scripts/Game/Client/State/ClientMainMenuState.cs [hunk 1]
```diff
@@ -59,7 +59,6 @@ async void InjectDependenciesAndInitialize(AuthenticationServiceFacade authServi
 
             if (string.IsNullOrEmpty(Application.cloudProjectId))
             {
-                PopupManager.ShowPopupPanel("Unity Gaming Services ProjectID not set up", "Click the Readme file in the Assets Folder within the Project window in-editor to follow \"How to set up Unity Gaming Services\"");
```
Comment (SamuelBellomo) [inline Assets/BossRoom/Scripts/Game/Client/State/ClientMainMenuState.cs:62]: Are there any plans to put this back somewhere? If users are doing the lobby flow, I'd assume they'd need to get that popup there if they haven't setup their project, no?

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +17,7 @@ feat: other players loading progress in loading screen [MTT-2239] (#580)
 feat: auto reconnect [MTT-2617] (#611)
 
 ### Changed
+fix: remove initial ugs popup [MTT-3563] (#650)
```
Comment (SamuelBellomo) [inline CHANGELOG.md:20]: Would a user understand this line reading it from cold without any context?

File: Assets/BossRoom/Scripts/Game/Client/State/ClientMainMenuState.cs (hunk 1 above)
Comment (LPLafontaineB) [reply Assets/BossRoom/Scripts/Game/Client/State/ClientMainMenuState.cs:62]: Good point! I replaced it with a tooltip instead, that shows up when hovering the disabled "Start with Lobby" button

File: Assets/BossRoom/Scripts/Game/Client/State/ClientMainMenuState.cs [hunk 3]
```diff
@@ -28,6 +28,7 @@ public class ClientMainMenuState : GameStateBehaviour
         [SerializeField] Button m_LobbyButton;
         [SerializeField] GameObject m_SignInSpinner;
         [SerializeField] UIProfileSelector m_UIProfileSelector;
+        [SerializeField] UITooltipDetector m_UGSSetupTooltipDetector;
```
Comment (SamuelBellomo) [inline Assets/BossRoom/Scripts/Game/Client/State/ClientMainMenuState.cs:31]: I tested this real quick. Text is really small and takes time to spawn when hovering. If I didn't know it was there, I could easily miss the tooltip.
Text should appear as soon as the mouse is hovering over the button and should adapt to the screen

Comment (LPLafontaineB) [reply Assets/BossRoom/Scripts/Game/Client/State/ClientMainMenuState.cs:31]: The delay is now removed for this tooltip and I made the text bigger

File: Assets/BossRoom/Scripts/Game/Client/UI/UITooltipDetector.cs [hunk 4]
```diff
@@ -27,9 +27,9 @@ public class UITooltipDetector : MonoBehaviour, IPointerEnterHandler, IPointerEx
         [Tooltip("Should the tooltip appear instantly if the player clicks this UI element?")]
         private bool m_ActivateOnClick = true;
 
-        // This delay-time could be a member variable, but it's annoying to have to change it on
-        // every tooltip spot everywhere! So for now we just hard-code it.
-        private const float k_TooltipDelay = 0.5f;
+        [SerializeField]
+        [Tooltip("The lenght of time the mouse needs to hover over this element before the tooltip appears (in seconds)")]
```
Comment (SamuelBellomo) [inline Assets/BossRoom/Scripts/Game/Client/UI/UITooltipDetector.cs:31]:
```suggestion
        [Tooltip("The length of time the mouse needs to hover over this element before the tooltip appears (in seconds)")]
```

**Review by SamuelBellomo — APPROVED** [review]

**Pull request merged**